.github/workflows/lint.yml

```diff
@@ -1,6 +1,6 @@
 name: Lint
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
What does this do?

Reply:
It adds a button allowing you to manually trigger a new build. This is something Travis CI has built in, but needs this tiny bit of config to enable for GHA.

https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
Like this:

![image](https://user-images.githubusercontent.com/1324225/148648434-303a78d7-d3de-4928-8e9d-51e656aca499.png)

Reply:
Thanks!